integration-tests/tests/billing-projects.js

```diff
@@ -79,26 +109,34 @@ const testBillingSpendReportFn = withUserToken(async ({ page, testUrl, token })
 
   // Interact with the Billing Page via mocked AJAX responses.
   const billingProjectName = 'OwnedBillingProject'
-  await setAjaxMockValues(page, billingProjectName, '90.13')
+  await setAjaxMockValues(page, billingProjectName, '1110')
```

Comment:
Updated the value to reflect the mock data (just to reduce confusion when reading the test).